test/CMakeLists.txt

```diff
@@ -86,7 +90,10 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR}/test)
 configure_file(gtest/env.h.in ${CMAKE_CURRENT_BINARY_DIR}/gtest/env.h)
 add_executable (gtestp4c ${GTESTP4C_SOURCES})
 target_link_libraries (gtestp4c ${GTEST_LDADD} ${P4C_LIBRARIES} gtest ${P4C_LIB_DEPS})
-add_dependencies(gtestp4c gtest genIR frontend controlplane)
+add_dependencies(gtestp4c gtest genIR frontend)
+if (ENABLE_CONTROL_PLANE)
+add_dependencies(gtestp4c controlplane)
```

Comment:
This likely not needed at all. Do we link in controlplane libs correctly? Overall all `add_dependencies` should be dropped, they just hide some underlying issues.